scaleway/provider.go

```diff
@@ -26,6 +33,19 @@ func DefaultProviderConfig() *ProviderConfig {
 
 // Provider returns a terraform.ResourceProvider.
 func Provider(config *ProviderConfig) plugin.ProviderFunc {
+	err := sentry.Init(sentry.ClientOptions{
+		Dsn:     dsn,
+		Release: version,
+	})
+	if err != nil {
+		log.Fatalf("sentry.Init: %s", err)
+	}
+	// Flush buffered events before the program terminates.
+	// Set the timeout to the maximum duration the program can afford to wait.
+	defer sentry.Flush(2 * time.Second)
```

Comment:
Do we want to recover from panic and send a sentry the same way we do in the CLI?